jujugui/static/gui/src/app/components/inspector/unit-list/test-unit-list.js

```diff
@@ -469,10 +512,11 @@ describe('UnitList', () => {
     var button = testUtils.findRenderedDOMComponentWithClass(
         output, 'button--neutral');
     testUtils.Simulate.click(button);
+    // Remove is still called to remove from ECS.
     assert.equal(destroyUnits.callCount, 1);
     assert.deepEqual(destroyUnits.args[0][0], [units[0].id, units[2].id]);
     // Make sure we mark the unit as resolved so that we can remove it.
-    assert.equal(envResolved.callCount, 2);
+    assert.equal(envResolved.callCount, 0);
```

Comment:
We were still making the calls in the test, but the API was stubbed, so this didn't show itself.